checksums.sh

```diff
@@ -17,17 +17,11 @@ fi
 SUBJECT_FOLDER=$1
 QSUB_FILE=".qsub.out.*"
 
-count=`ls -a 2>/dev/null | grep $QSUB_FILE | wc -l`
-if [ $count != 0 ]
-then
-    echo "###########################################"
-    echo "############ Checksum of docker image #####"
-    echo "###########################################"
-    grep "Digest: sha256:" $QSUB_FILE
-fi
+(ls ${QSUB_FILE} && grep "Digest: sha256:" ${QSUB_FILE}) || (die ${SUBJECT_FOLDER} "Cannot find sha256 digest of docker image")
```

Comment:
Here even if it is empty the banner would get printed. Is that the normal convention ?